ssh-keygen: refresh page (#9874)
tldr-pages/tldr: pages/common/ssh-keygen.md

`ssh-keygen`

> - Generate an ed25519 key with 100 key derivation function rounds and save the key to a specific file:

Reviewer: yes, that's actually clearer

> - Generate an ed25519 key with 100 key derivation function rounds and save the key to a specific file:

Reviewer 2: Actaully we can scrap this suggestion and go for one of the ones above. I missed them before, sorry!

https://github.com/tldr-pages/tldr/pull/9874#discussion_r1111217867

I think this is better.

> - Generate an ed25519 key with 100 key derivation function rounds and save the key to a specific file:

Author: The two proposals differ only in the word `specific`, right? However, saving to a file could also mean that you cannot freely choose the file name. With `specific` it sounds to me more like you can choose the filename. On the other hand, you can see from the command that you are saving it to a specific file.
As I understood it, the majority is now for the proposal without `specific`, right?

> - Generate an ed25519 key with 100 key derivation function rounds and specify the file in which to save the key:

Reviewer 3: I think this one is easier to read.
```suggestion
- Generate an ed25519 key with 100 key derivation function rounds and save the key to a file:
```

Reviewer 4: ```suggestion
- Generate an ed25519 key with 100 key derivation function rounds and save the key to a specific file:
```
How about this?

Reviewer 2: I'm more keen blueskuson's versions here.

Refer to: https://github.com/tldr-pages/tldr/pull/9874#discussion_r1162514532

- Change the type of the key format (for example from OPENSSH format to PEM), the file will be rewritten in-place:

`ssh-keygen -p -N "" -m {{PEM}} -f {{~/.ssh/OpenSSH_private_key}}`

- Retrieve public key from secret key:

> `ssh-keygen -y -f {{OpenSSH_private_key}}`

Reviewer 4: ```suggestion
`ssh-keygen -y -f {{~/.ssh/OpenSSH_private_key}}`
```
